composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "pos-lifestyle/laravel-nova-date-range-filter",
+    "description": "A Laravel Nova filter.",
+    "keywords": [
+        "laravel",
+        "nova",
+        "date",
+        "date range",
+        "filter"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": ">=7.1.0",
+        "laravel/framework": "^5.8|6.*",
+        "laravel/nova": "2.*"
+    },
+    "require-dev": {
+        "roave/security-advisories": "dev-master"
+    },
+    "autoload": {
+        "psr-4": {
+            "PosLifestyle\\DateRangeFilter\\": "src/"
```

Comment:
The namespace doesn't feel right.
The Namespace should be `pos-lifestyle` or `PosLifeStyle`or `Pos`
@Aeroek what do you think?

Reply:
1. `pos-lifestyle` is no valid namespace.
2. `PosLifeStyle` is not the correct spelling, because the company name is "POS lifestyle".
3. `Pos` is not unique and meaningful enough.